Expand docs on mass shapes

mayou36 opened the pull request.

apuignav commented on docs/usage.rst:
```diff
@@ -116,7 +116,11 @@ can be performed using normal python loops without loss in performance:
+are the `TensorFlow Probability distributions <https://www.tensorflow.org/probability/api_docs/python/tfp/distributions>`_
+or, for more customized mass shapes, the
+`zfit pdfs <https://zfit.github.io/zfit/model.html#tensor-sampling>`_ *(currently an
+experimental features is needed, contact the `zfit developers <https://github.com/zfit/zfit>`_ to learn more).*
```
features -> feature.

mayou36 commented:
Fixed and refactored the mass creator. Explicit casts and a broadcast (tfp dist broadcast automatically).
(yes, not every line is _strictly_ needed, but since user gonna copy paste the docs, this is the save way)
Ready from my side
